TIVar.cpp

```diff
@@ -261,73 +275,99 @@ int TIVar::stringToStrVar8x(String s, uint8_t* strVar, enum Endpoint model) {
 		tokenlen++;
 	}
 
-	TIVar::intToSizeWord(tokenlen, strVar);
+	if (type == STR_89) {
+		strVar[0] = '\0';
```

Comment:
I'm skeptical about the correctness of this section, because the length appears to be unspecified. Can you confirm that this is correct?